src/moscot/plotting/_utils.py

```diff
@@ -359,8 +359,8 @@ def _plot_temporal(
     adata: AnnData,
     temporal_key: str,
     key_stored: str,
-    start: float,
-    end: float,
+    source: float,
```

Comment:
Think `source/target` should be `Any`.

Reply:
In this case actually `float` because we expect `adata.obs[time_key]` to be of type `float`